### ISSUE TITLE
Cannot resolve bootstrap path in scss

### ISSUE THREAD

**Issue by kmergen** [coreui/coreui-free-bootstrap-admin-template#384]
This is a help request.

I use coreui as composer extension.

The problem is to resolve the ``node_modules`` path.

I use gulp-sass to build my css.

Here my file structure:
![image](https://user-images.githubusercontent.com/1640101/39513224-ee7999fa-4df3-11e8-9436-f63bbdd28bbb.png)

And here is my theme directory in my application from where I want to build my css:
![image](https://user-images.githubusercontent.com/1640101/39513343-50a87da8-4df4-11e8-91eb-6c199daf6913.png)

My style scss:
![image](https://user-images.githubusercontent.com/1640101/39513405-8425429c-4df4-11e8-820b-be96f9b41e98.png)

My gulpfile.js:
![image](https://user-images.githubusercontent.com/1640101/39513445-a4950ddc-4df4-11e8-86a5-1b8bf831426a.png)

To get it work I must change the bootstrap path as follows:
![image](https://user-images.githubusercontent.com/1640101/39513629-43208350-4df5-11e8-9bea-ea3b5b8b235e.png)

Is there a way to use gulp and not to change the path ?

- Operating system and version (Windows 10)
- Browser and version (Chrome latest)

**Comment by shurmanidze** [comment]
Right now I'm facing a similar issue. It's impossible to use coreui packages in monorepo with lerna or yarn workspaces. The node_modules folder can be several levels up with all dependencies hoisted. So there shouldn't be any direct references to the node_modules folder.

**Comment by luislezcair** [comment]
I'm facing this same issue with Yarn. CoreUI is importing bootstrap from the node_modules directory and it's not finding it. I think CoreUI should not be referencing bootstrap, it should be manually included by us because when deploying to production the node_modules directory gets overwritten and that change is lost.

**Comment by luislezcair** [comment]
I found a PR to fix this issue https://github.com/coreui/coreui/pull/4 but the original author seems to have deleted his fork.

**Comment by mrholek** [comment]
We don't want to duplicate Bootstrap's css styles, so we overwrite bootstrap variables and generate our own version of Bootstrap to keep CoreUI's styles as light as possible. That's why we're referencing to Bootstrap files.

**Comment by shurmanidze** [comment]
> That's why we're referencing to Bootstrap files.

Yes, but the question is not about 'why' it's about 'how'. The [PR](https://github.com/coreui/coreui/pull/4) mentioned about seems to be a solution. Anyway, on my side it works properly in the lerna/yarn environment without any extra dev dependencies. I just remove node_modules from the import path.

**Comment by luislezcair** [comment]
Yep, I also tried removing the node_modules reference from the import statements as a workaround. But if I deploy to Heroku for example, this change will get lost because Heroku will install a new copy. That's why I forked the repo, changed those imports there and then I installed with Yarn using my own copy.

**Comment by kazazes** [comment]
For anyone else with the same problem, I fixed it by modifying my root package.json and setting coreui and bootstrap to nohoist.

``"workspaces":` {
    "packages": [
      "packages/*"
    ],
    "nohoist": [
      "**/@coreui/**",
      "**/bootstrap/**"
    ]
  },``

**Comment by jovanmaric** [comment]
> I found a PR to fix this issue [coreui/coreui#4](https://github.com/coreui/coreui/pull/4) but the original author seems to have deleted his fork.

I totally forgot about this merge request and thought that I could remove the original repo. I will try to restore it for what it's worth.

**Comment by luisamor** [comment]
I have the same issue:
in my local style.scss:
```
// If you want to override variables do it here
@import "variables";

// Import styles
@import "../node_modules/@coreui/coreui/scss/coreui";

// If you want to add something do it here
@import "custom";
```
and can't find node_modules
![image](https://user-images.githubusercontent.com/11832980/47577943-6b8bd900-d948-11e8-87f5-58f53ceb78ae.png)
